ReplaysAndMods.hpp

```diff
@@ -175,9 +175,9 @@ namespace ReplaysAndMods {
 		constexpr auto modInfoSize = std::size_t{22};
 		auto modInfo = std::string{modInfoSize, {}, std::string::allocator_type{}};
 		copyFixed(replay, modInfo.begin(), modInfo.size());
-		auto modVersion = modInfo.substr(modInfo.find_last_of('\0', modInfo.find_last_not_of('\0')) + 1);
-		modInfo.resize(std::min(modInfo.size(), modInfo.find('\0')));
-		modVersion.resize(std::min(modVersion.size(), modVersion.find('\0')));
+		auto modVersion = modInfo.substr(modInfo.find_last_of(L'\0', modInfo.find_last_not_of(L'\0')) + 1);

```

Comment:
modInfo is a `std::string`, so here `char` (`'\0`) should be used instead of `wchar_t` (`L'\0'`).

Reply:
This is definitely a fault by simply replacing all `'\0'` to `wchar`.